### ISSUE TITLE
Wall Texture Perspective Bug

### ISSUE THREAD

**Issue by 0xABADCAFE** [mheyer32/alienbreed3d2#40]
This ties in with the other issues regarding the swimming floor, etc. While working on a mod, I created an area with precise geometry to represent boxes using 1:1 texture scaling (no stretch or compress to fit). It seems that the wall textures have a different (lower) horizon than the geometry. As you back away from a wall, the vertical texture coordinates descend slightly:
![wall2](https://user-images.githubusercontent.com/27722116/208096922-1f24bf2c-0de4-4058-8ba2-e5092aa8f3bd.png)
![wall1](https://user-images.githubusercontent.com/27722116/208096928-6c5b62ff-e1ac-4be5-b39c-02ea91e65a04.png)
![wall2](https://user-images.githubusercontent.com/27722116/208096938-e7be09e9-ec36-47ee-a268-9076487eee08.png)
![wall1](https://user-images.githubusercontent.com/27722116/208096942-08d0dc54-ff13-4b11-901a-2890622005e3.png)

In the second image, the bottom of the brown container texture is clearly visible at the top edge.

**Comment by 0xABADCAFE** [comment]
I may have just fixed this based on a previous suggestion by Andy Loft. He suggested on EAB redefining FS_HEIGHT to 240, but then using FS_HEIGHT-8 in a few places later. I only made FS_HEIGHT 240 and in the code as it stands, that seems to have fixed the issue:
![walls3](https://user-images.githubusercontent.com/27722116/208106440-96d899b8-8c09-4405-b467-fa4d211a7c19.png)
![walls4](https://user-images.githubusercontent.com/27722116/208106450-2ba809be-7523-4d7f-b34d-2729ad39b18b.png)

**Comment by mheyer32** [comment]
With the current change of the rendering height to 240, the discrepancies between wall and floor rendering have been papered over, but we do now render a few more lines in the chunky buffer that the player will never see as we don't copy them over into the visible buffer during C2P.

I think there should be an engine-friendly way to get around this: initialize the vertical clip variable with 232 instead of 240 :-)
I _think_ this is the same mechanism that letterbox rendering uses when decreasing the size of the viewport.

**Comment by 0xABADCAFE** [comment]
I totally agree. I just felt the cosmetic fix was worth it for the UX. No intention of closing the ticket until it's fixed correctly.

**Comment by 0xABADCAFE** [comment]
I think this bug can be closed now that the visible render area was increased to 240.
